sudo/views.py

```diff
@@ -88,10 +89,14 @@ def sudo(request, template_name='sudo/sudo.html', extra_context=None):
     if request.is_sudo():
         return HttpResponseRedirect(redirect_to)
 
+    if request.method == 'GET':
+        request.session['redirect_to'] = redirect_to
```

Comment:
Let's rename this and move it to a constant so we don't conflict. Let's use:

```python
REDIRECT_TO_FIELD_NAME = 'sudo_redirect_to'
```

If you think it makes sense, we could even expose this to a configurable setting, like, `SUDO_REDIRECT_TO_FIELD_NAME`

Thoughts?